src/coins.h

```diff
@@ -397,6 +398,17 @@ class CCoinsView
     //! Get current history root
     virtual uint256 GetHistoryRoot(uint32_t epochId) const = 0;
 
+    //! Get the largest completed subtree data for the TRACKED_SUBTREE_HEIGHT subtrees known
+    //! to the node for a given protocol. std::nullopt is returned in the event there are no
+    //! complete subtrees.
+    virtual std::optional<libzcash::LatestSubtree> GetLatestSubtree(ShieldedType type) const = 0;
+
+    //! Gets the cached data about the TRACKED_SUBTREE_HEIGHT subtree for the specified
+    //! protocol at the provided index, if that subtree is complete.
+    virtual std::optional<libzcash::SubtreeData> GetSubtreeData(
+            ShieldedType type,
+            libzcash::SubtreeIndex index) const = 0;
+
     //! Do a bulk modification (multiple CCoins changes + BestBlock change).
     //! The passed mapCoins can be modified.
```

Comment:
Document what other objects passed to this method are modified. (E.g. `cacheSaplingSubtrees` and `cacheOrchardSubtrees` are cleared.